dist/flags.py

```diff
@@ -136,12 +138,12 @@
         'SESSION_LOCKED_TURTLE',
         'SESSION_LOGGING_INMEM',
         'SESSION_LOOKASIDE_CURSOR',
-        'SESSION_NO_CACHE',
         'SESSION_NO_DATA_HANDLES',
-        'SESSION_NO_EVICTION',
+        'SESSION_NO_RECONCILE',
         'SESSION_NO_LOGGING',
         'SESSION_NO_SCHEMA_LOCK',
         'SESSION_QUIET_CORRUPT_FILE',
+        'SESSION_READ_WONT_NEED',
```

Comment:
I'm happy to talk through flag naming: we had `NO_EVICTION` sprinkled liberally through the codebase, leading to many paths ignoring the cache size and unable to split trees normally.  This renaming attempts to clarify what happens when `NO_EVICTION` is set.